POS/Operation/03_EditBillList.md

```diff
@@ -226,20 +220,23 @@ If you want to assign purchase to a customer who does not yet exist in the syste
     ![Create Customer](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/AddCustomer02.png "[Create Customer]")
 
 3. Enter the customer data in the corresponding fields and click the [ ![Create Customer](../../Assets/Icons/Customer01.png "[Create Customer]") Create Customer] button.   
-    The customer is registered. The customer data view is closed. The bill list is displayed again. The created customer name and its address is displayed in the upper left corner of the bill list.
+    The customer is registered. The customer data view is closed. The bill list is displayed again.   
+    The customer's name and address are displayed on top of the bill list. The data is also stored in the *Order Management* module. <!--- Passiert das automatisch oder muss man da noch was tun?-->
 
     > [Info] If you want to unassign the customer from the purchase, click the ![Delete](../../Assets/Icons/Trash01.png "[Delete]") (Delete) button in the upper right corner of the bill list.
 
 
 
 ### Assign a purchase to an existing customer
 
-Assign the current purchase to an registered customer.
+Assign the current purchase to a registered customer.   
+
+Alternatively, if the customer has a customer or member card, you can select the customer via customer number, see [Assign a purchase to an existing customer via customer number](#assign-a-purchase-to-an-existing-customer-via-customer-number).
```

Comment:
[Assign a purchase to an existing customer via customer number](#assign-a-purchase-to-an-existing-customer-via-customer-number) --> Achtung, Link stimmt nicht: Es gibt zwei Procedures: Assign a purchase to an existing customer und Assign a purchase via customer number. Check!